javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -2,7 +2,7 @@
 function addSuffixToAll(words, suffix) {
   var suffixArray = [];
   for (var i = 0; i < words.length; i++) {
-    suffixArray.push(words[i] += suffix);
+    suffixArray[i] = (words[i] += suffix);
```

Comment:
The `+=` is still making a change to the original array. Make sure you are verifying that you fixed bugs before re-submitting.